app/javascript/application.js

```diff
@@ -1,5 +1,6 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import '@hotwired/turbo-rails'
+import 'controllers'
```

Comment:
I think these belong under datatables/stupidtable, because they're not libraries, but app-specific files.